setup.py

```diff
@@ -19,4 +26,16 @@
     data_files=[
         ('share/metainfo', ['com.github.whipper_team.Whipper.metainfo.xml']),
     ],
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Intended Audience :: End Users/Desktop',
+        'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
+        'Natural Language :: English',
+        'Operating System :: POSIX :: Linux',
```

Comment:
Do we know if it works under any other OS’s? Do we support it running under any non‐Linux system?

Reply:
It might work just fine on *BSD - worth checking if anyone uses it.

Reply:
It’s easy to add later though, and maybe we also shouldn’t add platforms that we’re not actively testing (and thus supporting)?